plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -204,17 +265,23 @@ class Bazel(
 
     /**
      * Get the packages for the dependencies of the given [scopes] using the given [registry], for the dependencies that
-     * do not have a local path override. The dependencies having an override defined in [localPathOverrides] are
-     * created without querying [registry] and will use [projectVcs] as VCS information.
+     * do not have a local path override. The dependencies having a local path override defined in [localPathOverrides]
+     * are created without querying [registry] and will use [projectVcs] as VCS information. The dependencies having an
+     * archive override defined in [archiveOverrides] are created without querying [registry] and will have their
+     * version suppressed.
      */
     private fun getPackages(
         scopes: Set<Scope>,
         registry: BazelModuleRegistryService,
         localPathOverrides: Map<String, String>,
+        archiveOverrides: Map<String, ArchiveOverride>,
         projectVcs: VcsInfo
     ): Set<Package> {
         val ids = scopes.collectDependencies()
-        val (packageIdsWithPathOverride, otherPackageIds) = ids.partition { it.name in localPathOverrides }
+        // Fortunately, two partitions can be done in a row since it can safely be assumed that a package cannot have
```

Comment:
Let's have an empty line before this one to group the comment to the below code.